src/app/services/graph-constants.ts

```diff
@@ -1,8 +1,9 @@
-export const USER_INFO_URL = 'https://graph.microsoft.com/v1.0/me';
-export const USER_PICTURE_URL = 'https://graph.microsoft.com/v1.0/me/photo/$value';
+export const GRAPH_URL = 'https://graph.microsoft.com';
+export const USER_INFO_URL = `${GRAPH_URL}'/v1.0/me'`;
```

Comment:
This will evaluate to an invalid string. Remove the single quotes for it to evaluate to a valid string.